docs/visualization/plotting-1d-data.ipynb

```diff
@@ -128,7 +130,7 @@
    "source": [
     "### Overplotting using a dict of DataArrays\n",
     "\n",
-    "One can also supply the `plot` function with a `dict` of data arrays, and compatible data arrays will be overplotted on the same axes:"
+    "One can also supply the `plot` function with a `dict` of data arrays, which will then be overplotted on the same axes."
```

Comment:
Why the change? We had a case yesterday where mismatching units led to the use of different axes, so I think the original was more correct? Maybe just clarify the original if unclear?